include/zephyr/zbus/zbus.h

```diff
@@ -74,6 +74,10 @@ struct zbus_channel {
 #if defined(CONFIG_ZBUS_CHANNEL_NAME) || defined(__DOXYGEN__)
 	/** Channel name. */
 	const char *const name;
+#endif
+#if defined(CONFIG_ZBUS_CHANNEL_ID) || defined(__DOXYGEN__)
+	/** Unique numeric channel identifier. */
+	uint32_t id;
 #endif
```

Comment:
Please, add a line before and after to keep consistency. 
```suggestion

#if defined(CONFIG_ZBUS_CHANNEL_ID) || defined(__DOXYGEN__)
	/** Unique numeric channel identifier. */
	uint32_t id;
#endif

```